### PR TITLE
[stable/superset] Add env vars from values or secret

### LINKED ISSUE #5736: [stable/superset] Unable to set env vars
**Is this a request for help?**: NO

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): FEATURE REQUEST

**Version of Helm and Kubernetes**: Any

**Which chart**: stable/superset

**What happened**: It's impossible to set env vars to use them in the config file.

**What you expected to happen**: To be able to set env vars from secret, configMap, value file

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by SharpEdgeMarshall**
**What this PR does / why we need it**: Add ability to set env vars from values or secret

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #5736 

**Special notes for your reviewer**: @alitari @mattjmcnaughton 

Maybe we should read SECRET_KEY from an env var by default?

### CODE REVIEW

**Review by alitari — COMMENTED** [review]
please update the README for the new configuration options as well

**Comment by SharpEdgeMarshall** [comment]
Sorry, I forgot to update the README, now should be ok.

File: stable/superset/values.yaml [hunk 1]
```diff
@@ -47,6 +47,14 @@ configFile: |-
   # Set this API key to enable Mapbox visualizations
   MAPBOX_API_KEY = ''
 
+## Extra environment variables that will be passed onto deployment pod
+##
+env: {}
```
Comment (mattjmcnaughton) [inline stable/superset/values.yaml]: I think that the "best practice" name for this variable would be `extraEnv` (and `extraEnvFromSecret`). My understanding is that the `charts` repo is trying to use `extraEnv` going forward.

cc @timstoop who first brought this to my attention :)

**Comment by SharpEdgeMarshall** [comment]
Fixed

**Comment by SharpEdgeMarshall** [comment]
Conflict solved

**Pull request merged**